Source/spells.cpp

```diff
@@ -147,6 +147,10 @@ int GetManaAmount(const Player &player, SpellID sn)
 
 void ConsumeSpell(Player &player, SpellID sn)
 {
+	Monster &golem = Monsters[player.getId()];

```

Comment:
Would it make sense to create a Player class method to get the player's golem here?

Reply:
Yes!